ui/admin/app/components/form/host/static/index.hbs

```diff
@@ -56,7 +56,7 @@
   <form.input
     @name='address'
     @type='text'
-    @value={{@model.attributes.address}}
+    @value={{@model.address}}
     @label={{t 'form.address.label'}}
     @error={{@model.errors.name}}
```

Comment:
On an unrelated note, shouldn't this be `errors.address`?

Reply:
it should be! good catch. I'm gonna update this pr

Reply:
@gsusmi  done!